### PR TITLE
make check-zzz-fluentd-prometheus-scrape.sh rely on oc exec

### PR DESCRIPTION
**Pull request by jcantrill**
Update check prometheus test to rely on oc exec

### CODE REVIEW

File: hack/testing/check-zzz-fluentd-prometheus-scrape.sh [hunk 1]
```diff
@@ -18,7 +18,6 @@ trap "cleanup" EXIT
 LOGGING_NS=${LOGGING_NS:-openshift-logging}
 
 fpod="$(get_running_pod fluentd)"
-fpod_ip="$(oc get pod ${fpod} -o jsonpath='{.status.podIP}')"
 
-os::cmd::try_until_success "curl -k https://${fpod_ip}:24231/metrics"
-curl -k https://${fpod_ip}:24231/metrics >> $ARTIFACT_DIR/${fpod}-metrics-scrape 2>&1 || :
+os::cmd::try_until_success "oc exec $fpod curl -k https://localhost:24231/metrics"
```
Comment (vimalk78) [inline hack/testing/check-zzz-fluentd-prometheus-scrape.sh]: need `--` before `curl` ?

**Comment by ewolinetz** [comment]
@jcantrill needs a rebase

**Comment by jcantrill** [comment]
Closing as this test was moved to CLO https://github.com/openshift/cluster-logging-operator/blob/master/hack/testing/test-999-fluentd-prometheus-metrics.sh

**Pull request closed without merging**